src/ir/module-splitting.cpp

```diff
@@ -860,64 +926,76 @@ void ModuleSplitter::shareImportableItems() {
       Name exportName = Names::getValidExportName(primary, baseName);
       primary.addExport(new Export(exportName, kind, primaryItem.name));
       secondaryItem.base = exportName;
+      exports[std::make_pair(kind, primaryItem.name)] = exportName;
```

Comment:
Is this a fix to avoid unnecessary duplicate exports? If so, it might be nice to separate that out to reduce test churn in this PR.